### ISSUE TITLE
Registry: does YAML make sense as a language?

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#4495]
@svrnm - does it really make sense for the language to be YAML here?

_Originally posted by @chalin in https://github.com/open-telemetry/opentelemetry.io/pull/4139#discussion_r1596022907_

**Comment by svrnm** [comment]
Good catch, I just wonder what the right language would be in that case?

**Comment by chalin** [comment]
I was wondering the same thing! 😄

**Comment by svrnm** [comment]
cc @ishanjainn 

GitHub tags ansible repositories with `python`
